variables.tf

```diff
@@ -46,6 +46,21 @@ variable "elasticache_subnets" {
   default     = []
 }
 
+variable "create_database_route_table" {
+  description = "Controls if saperate route table for database should be created"
```

Comment:
`separate` - check it everywhere. Also, I think `database subnet` is more correct - add word "subnet" to all variables.

Reply:
Added this to all variables now.